src/ui/Logic/Language.cs

```diff
@@ -2477,7 +2477,6 @@ public Language()
                 ExactlyTwoLines = "Exactly two lines",
                 MoreThanTwoLines = "More than two lines",
                 Bookmarked = "Bookmarked",
-                BookmarkContains = "Bookmark contains",
```

Comment:
Merge error?

Reply:
> Merge error?

I did not changed this file ?  Strange i added the line again.